helpers/html.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+require 'sinatra/base'
+
+module Sinatra
+  module HTMLEscapeHelper
+    def h(text)
```

Comment:
Is `h` a general standard for naming functions like this? I'm generally a fan of brevity, but I'm a little hesitant here, unless this is a universally recognised approach.